### PR TITLE
 {bio} [foss-2015b,goolf-1.7.20] New Tophat version (REVIEW)

### PR DESCRIPTION
**Pull request by rjeschmi**
They finally fixed SeqAn to work with newer compilers so this should work fine with up to date foss and goolf-1.7.20

will test more before putting in review.

### CODE REVIEW

File: easybuild/easyconfigs/t/TopHat/TopHat-2.1.1-goolf-1.7.20.eb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['http://ccb.jhu.edu/software/tophat/downloads/']
+
+patches = [
+    'tophat-2.0.13-zlib.patch',
+]
+
+dependencies = [
+    ('Boost', '1.58.0', '-Python-2.7.10'),
```
Comment (boegel) [inline easybuild/easyconfigs/t/TopHat/TopHat-2.1.1-goolf-1.7.20.eb]: 2.7.11 :)

Comment (boegel) [inline easybuild/easyconfigs/t/TopHat/TopHat-2.1.1-goolf-1.7.20.eb]: and maybe Boost 1.60?

File: easybuild/easyconfigs/t/TopHat/TopHat-2.1.1-foss-2015b.eb [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['http://ccb.jhu.edu/software/tophat/downloads/']
+
+patches = [
+    'tophat-2.0.13-zlib.patch',
+]
+
+dependencies = [
+    ('Boost', '1.59.0', '-Python-2.7.10'),
```
Comment (boegel) [inline easybuild/easyconfigs/t/TopHat/TopHat-2.1.1-foss-2015b.eb]: Boost 1.60 + Python 2.7.11?

Comment (rjeschmi) [inline easybuild/easyconfigs/t/TopHat/TopHat-2.1.1-foss-2015b.eb]: I'd rather keep the deps the same within the toolchain.

I'll do a foss-2016a version with the bumps

**Comment by verdurin** [comment]
Ah - hadn't spotted this one. 

Shall I delete the one I started at #2586 ?

**Comment by boegel** [comment]
@verdurin: well, let's figure out which one we're going with first...

@rjeschmi: thoughts on #2586?

**Comment by boegel** [comment]
@verdurin, @rjeschmi : any updates on which PR 'wins'? ;)

**Comment by verdurin** [comment]
For the sake of progress, I'm happy to go with @rjeschmi, especially given that he covers two toolchains.

**Comment by rjeschmi** [comment]
I'll pull boost up for the foss-2015b version.

I'd like to build it with a newer version of GCC, but we dont' really have a test.

This will make it functionally the same to @verdurin for foss-2015b and will still have a working version for goolf-1.7.20

This is a maintenance release of TopHat and they recommend using HISAT instead now so we might as well get it fright.

**Comment by boegel** [comment]
@rjeschmi ping?

**Comment by rjeschmi** [comment]
Sorry, bugs in the network here have made it really annoying to try and fix this. Will work on it today.

**Comment by rjeschmi** [comment]
This should be sufficient to cover both mine and @verdurin

File: easybuild/easyconfigs/t/TopHat/TopHat-2.1.1-goolf-1.7.20.eb (hunk 1 above)
Comment (boegel) [inline easybuild/easyconfigs/t/TopHat/TopHat-2.1.1-goolf-1.7.20.eb]: @rjeschmi do you want to stick to Boost 1.58.0 here, or shall we bump this to 1.60.0 too to have both TopHat easyconfigs in sync?

**Comment by boegel** [comment]
@rjeschmi If this is no longer WIP, please adjust the PR title accordingly

File: easybuild/easyconfigs/t/TopHat/TopHat-2.1.1-goolf-1.7.20.eb (hunk 1 above)
Comment (rjeschmi) [inline easybuild/easyconfigs/t/TopHat/TopHat-2.1.1-goolf-1.7.20.eb]: I moved it up in the newer toolchain version. 1.58.0 is the supported version for 1.7.20 and the hpcbios "release" where we agreed to 1.7.20.

Comment (boegel) [inline easybuild/easyconfigs/t/TopHat/TopHat-2.1.1-goolf-1.7.20.eb]: OK, fine :)

**Comment by boegel** [comment]
@rjeschmi with #2586 being merged, this needs a resync with `develop`...

**Comment by boegel** [comment]
@rjeschmi ping?

**Comment by rjeschmi** [comment]
I don't think we really need to merge this.

**Pull request closed without merging**